pyomo/contrib/mindtpy/MindtPy.py

```diff
@@ -363,6 +372,14 @@ def solve(self, model, **kwds):
             solve_data.results.problem.lower_bound = solve_data.LB
             solve_data.results.problem.upper_bound = solve_data.UB
 
+        solve_data.results.solver.timing = solve_data.timing
+        solve_data.results.solver.user_time = solve_data.timing.total
+        solve_data.results.solver.wallclock_time = solve_data.timing.total
+
+        solve_data.results.solver.iterations = solve_data.mip_iter  # TODO @David this makes sense right?
```

Comment:
Yeah, we usually report the master iterations. You'll want to double-check when the mip_iter advancement happens to make sure that `mip_iter += 1` doesn't happen at an inconvenient location; otherwise, you'll always report one more iteration than it actually required.

Reply:
Fixed in 0afb2f58e8f91bd681aeeb5213bcc069392e80a7